app/src/main/java/com/eventyay/organizer/core/event/dashboard/EventDashboardFragment.java

```diff
@@ -208,7 +208,11 @@ public void showOrderStatistics(OrderStatistics orderStatistics) {
 
     @Override
     public void showError(String error) {
-        ViewUtils.showSnackbar(binding.getRoot(), error);
+        if (error.contains("starts-at")){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['{' is not preceded with whitespace.](https://app.codacy.com/manual/fossasia/open-event-orga-app/pullRequest?prid=4963535)